src/Person/Person.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.css';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => {
+  const ispartner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <div className="person">
+      <h2 className="person__name">
+        My name is
+        {' '}
+        {name}
+      </h2>
+      <p className="person__age">
+        {age && (`I am ${age}`) }
+      </p>
```

Comment:
If you don't get age, will you render empty p? Can you fix that?

Reply:
If you don't get age, you will render p with 0 text (default prop is 0):
<p className="person__age">0</p>

Is it actually what you wanted?